### DISCUSSION TITLE
Notes for March 21st meeting

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#83] [General]
#### Specification

* No further remarks on [first batch](https://github.com/linked-statistics/COOS/commit/8fe20fca2c49ae7a6bfaebb21c5c48cf70e7a106) of comments from Thérèse.
* Minor editorial comments from Flavio on second batch, then [pull request](https://github.com/linked-statistics/COOS/pull/80) accepted and issue #70 closed.
* No remarks on Flavio's [suggestions](https://github.com/linked-statistics/COOS/issues/42) for introduction / conclusion (but addition requested, see below) --> update specification document

#### Issues

* Discussion on issue #59 going on
* Discussion on issue #65 going on
* See Jenny's remark about LinkedIn in issue #76
* Remaining points in issue #42 to discuss

### Follow-up on action points

* Add a sentence explaining about all models (as a context) in the beginning of “background” sub-section: who can provide the sentence?
* On issue #78, waiting for conclusions of the [discussion](https://github.com/UNECE/GSIMRevision/issues/3) in the GSIM revision task team
* Also on issue #78, we have to find a name for the property used to reference a GSBPM phase or sub-process from a standard (or reuse existing `rdfs:seeAlso` or other) 
* [Abstract](https://github.com/linked-statistics/COOS/discussions/82) for ModernStats World Workshop submitted (and accepted)

### Other

* Rosemary's [feedback](https://github.com/linked-statistics/COOS/issues/72)
* Jay's [feedback](https://github.com/linked-statistics/COOS/issues/69)

**Comment by ChLaaboudi** [comment]
Potential interesting event:
https://op.europa.eu/en/web/endorse/follow-up-events 
The next follow-up event of ENDORSE will take place on 30 March 2022 (10.00-12.00 CET) as an online webinar on Corporate services on reference data management - VocBench and ShowVoc in practice.
